card/lib/cardio.rb

```
@@ -8,7 +8,11 @@
 module Cardio
 
   ActiveSupport.on_load :card do
-    Card::Loader.load_mods if Card.count > 0
+    if Card.count > 0
+      Card::Loader.load_mods 
+    else
+      Rails.logger.warn "empty database"
```

I wasn't sure whether we want to raise an exception here